stories/icon.stories.ts

```diff
@@ -10,39 +10,38 @@ import { all } from '@igniteui/material-icons-extended';
 
 const icons = all.map((icon) => icon.name);
 
+// region default
 export default {
   title: 'Icon',
   component: 'igc-icon',
   argTypes: {
     name: {
+      description: 'The name of the icon glyph to draw.',
+      defaultValue: '',
       control: {
         type: 'select',
         options: icons,
       },
-      defaultValue: 'biking',
-      description: 'Name of the icon',
     },
     collection: {
-      control: {
-        type: 'text',
-      },
+      description:
+        'The name of the registered collection for look up of icons.\nDefaults to `default`.',
       defaultValue: 'default',
-      description: 'Collection of icons',
-    },
-    size: {
-      control: {
-        type: 'inline-radio',
-        options: ['small', 'medium', 'large'],
-      },
-      defaultValue: 'medium',
+      control: 'text',
     },
     mirrored: {
+      description: 'Whether to flip the icon. Useful for RTL layouts.',
+      defaultValue: 'false',
       control: 'boolean',
-      defaultValue: false,
+    },
+    size: {
+      description: 'Determines the size of the component.',
+      defaultValue: 'medium',
+      control: 'text',
```

Comment:
Size is incorrectly generated as string/text. This is caused by the changing of the default size in the `IgcIcon` constructor. However I couldn't find a way to fix it.